docs/developer_guide/conversion.md

```diff
@@ -4,4 +4,81 @@ title: Model Conversion Reference
 
 !!! warning
 
-    Looking to implement your own converter? This reference is on the way. Stay tuned!
+    This reference is a work in progress. Stay tuned for new updates!
+
+This reference guide describes all there is to know about Fast-LLM's checkpoint conversion system.
+After reading this, you should be able to create your own `External` converter, in Hugging Face format or other.
+And if you are familiar with the rest of Fast-LLM, you will also be able to create an entirely custom converter.
+
+## Conversion and checkpointing basics
```

Comment:
there should be somewhere an explanation that Fast-LLM distinguishes internal and external checkpoint formats/handlers, and that this guide mostly covers external ones (with examples, i.e. "huggingface", "onnx", so on).

Reply:
Isn't this covered in the intro and first section?